lib/web_ui/dev/common.dart

```diff
@@ -150,12 +161,16 @@ class _MacBinding implements PlatformBinding {
 
   @override
   String getFirefoxDownloadUrl(String version) =>
-      'https://download-installer.cdn.mozilla.net/pub/firefox/releases/${version}/mac/en-US/firefox-${version}.dmg';
+    'https://download-installer.cdn.mozilla.net/pub/firefox/releases/${version}/mac/en-US/'
+        '${getFirefoxDownloadFilename(version)}';
 
   @override
-  String getFirefoxExecutablePath(io.Directory versionDir) {
-    throw UnimplementedError();
-  }
+  String getFirefoxDownloadFilename(String version) =>
+      'Firefox ${version}.dmg';
```

Comment:
question: is there a space between name and version?

Reply:
Yes